src/psyclone/psyir/backend/sympy_writer.py

```diff
@@ -52,11 +55,37 @@ class SymPyWriter(FortranWriter):
     implements special handling for constants (which can have a precision
     attached, e.g. 2_4) and some intrinsic functions (e.g. MAX, which SymPy
     expects to be Max).
+    It additionally supports accesses to structure types. A full description
+    can be found in the manual:
+    https://psyclone-dev.readthedocs.io/en/latest/sympy.html#internal-details
 
     '''
 
-    def __init__(self):
+    def __init__(self, list_of_expressions=None):
```

Comment:
Please document the `list_of_expressions` argument. I'm wondering whether it would be nicer to have it as a `list_of_references` as, in use, it looks a bit odd to construct the SymPyWriter and give it a list of expressions, only to then pass those same expressions to the writer to get the generated string.

Reply:
Done (though that argument is now elsewhere :) ).